src/Phoenix.vue

```diff
@@ -7,8 +7,20 @@
       </template>
       <template v-else>
         <message-bar :active-messages="activeMessages" @deleteMessage="$_deleteMessage" />
-        <top-bar :applicationsList="$_applicationsList" :activeNotifications="activeNotifications" :user-id="user.id" :user-display-name="user.displayname" :hasAppNavigation="!!appNavigationEntries.length" @toggleAppNavigation="$_toggleAppNavigation(!appNavigationVisible)"></top-bar>
-        <side-menu :visible="appNavigationVisible" :entries="appNavigationEntries" @closed="$_toggleAppNavigation(false)"></side-menu>
+        <top-bar
+          :applicationsList="$_applicationsList"
+          :activeNotifications="activeNotifications"
+          :user-id="user.id"
+          :user-display-name="user.displayname"
+          :hasAppNavigation="appNavigationEntries.length > 1"
+          @toggleAppNavigation="$_toggleAppNavigation(!appNavigationVisible)"
+        />
+        <side-menu
+          v-if="appNavigationEntries.length > 1"
+          :visible="appNavigationVisible"
+          :entries="appNavigationEntries"
+          @closed="$_toggleAppNavigation(false)"
```

Comment:
Ok, since you set an explicit value here, I'd vote for `$_setAppNavigationVisible` as function name.

Reply:
I went a bit different direction. Since this event is responsible only for closing I went with method called `hideAppNavigation` which doesn't take any argument. Hope that's okay :wink:

Reply:
perfect 😍